apollo-api/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ kotlin {
     val commonMain by getting {
       dependencies {
         api(groovy.util.Eval.x(project, "x.dep.okio"))
+        api(groovy.util.Eval.x(project, "x.dep.uuid"))
```

Comment:
This is used to create UUIDs that are later used in `multipart/form-data` HTTP bodies. I think this is a small enough dependency that we can add it to `apollo-api`. If we wanted, it should be relatively easy to use a random string instead.